applying-stpa/l1_reqs.txt

```diff
@@ -0,0 +1,228 @@
+REQ-1: The OpenAPS rig must pull ___ data from the other components of the system, every X seconds or whenever this data is available
```

Comment:
Please specify what should be in the "pull ____ data", and clarify the time period for "X"

Reply:
This is the one thing I have left, there are a few mentions of X so this is a bigger task